### DISCUSSION TITLE
Support for ->live() ?

### DISCUSSION THREAD

**Discussion by sinnbeck** [awcodes/filament-tiptap-editor#155] [Q&A] [answered]
Does this plugin support the `->live()` method? I cannot seem to get it to work. Running 3.0.0-beta-1
```
TiptapEditor::make('body')
      ->profile('minimal')
      ->live()
      ->required(),

**Comment by awcodes** [comment]
You can try the latest beta. It should work, but I want to encourage you to do everything possible to not use live() on any rich text editor. The consequences are dire. You don't want server requests being made for every character typed into any field that allows long form content or constant state updates. It will be horrible for your users.

**Reply by sinnbeck** [reply]
Yeah I know and normally I wouldn't. But the system I am building is only going to have one user and I am adding a bit of live preview of the expected output. Normally I would use alpine but I don't see any of doing so with filament (between components)

But thanks for the heads up 😊

**Reply by sinnbeck** [reply]
I am running 3.0.0-beta-1 which seems to be the newest version, but it sadly does not work

**Reply by awcodes** [reply, marked as answer]
After some thought. Yea. Live isn't going to work in 3 because there is no input and reactivity like that depends on the event listeners fired from inputs.

**Reply by awcodes** [reply]
I also recommend the Peek plugin for content previews.

**Reply by sinnbeck** [reply]
Yeah I rewrote it to just have a preview button that opens a modal with the data. I will take a look at peek, but it might be overkill. It is just a simple email preview (replacing "mentions"), so I just wrote my own form html preview component

**Reply by pboivin** [reply]
Hey @sinnbeck, [Peek](https://github.com/pboivin/filament-peek) has built-in support for Tiptap. If you're already far along with your custom previewer then this won't save you much time. But just in case, here's a quick demo of the kind of integration that is currently possible:

https://github.com/awcodes/filament-tiptap-editor/assets/7805679/94568089-1b8c-4cf8-9b5e-923b4b9aaebe

**Reply by sinnbeck** [reply]
@pboivin Thanks for the video. It indeed looks awesome. My own version is really simple as it's just the html from the rich editor that is rendered (after mentions are replaced). I might give it a shot anyways 👍
